feedback/lib/src/l18n/translation.dart

```diff
@@ -138,6 +174,8 @@ class GlobalFeedbackLocalizationsDelegate
     const Locale('de'): const DeFeedbackLocalizations(),
     const Locale('fr'): const FrFeedbackLocalizations(),
     const Locale('ar'): const ArFeedbackLocalizations(),
+    const Locale('ru'): const RuFeedbackLocalizations(),
+    const Locale('uk'): const UkFeedbackLocalizations(),
```

Comment:
@volodymyrd Do you mind double-checking this? Sometimes it's `uk` and sometimes it's `ua`. What is the more correct one?

Reply:
yes, sure, please take a look at:
uk  - it's for the [language code](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=uk)
UA - it's for the [country code](https://en.wikipedia.org/wiki/ISO_3166-2:UA) 
full local is uk_UA, but your app uses only a language code, so uk is correct one

Reply:
I see, thanks for the clarification.